source/marineRegions.ts

```diff
@@ -1,185 +1,189 @@
-import { Sequence } from "@cognite/sdk";
```

Comment:
Reformatting and type improvements. This will be deprecated in the next version.